### PR TITLE
Data-ajax - Improved the documentation to explain the updated event

### PR DESCRIPTION
**Pull request by duboisp**
Docs and showing an example on how to use "wb-contentupdated" event

### CODE REVIEW

File: site/pages/docs/ref/data-ajax/data-ajax-en.hbs [hunk 1]
```diff
@@ -9,13 +9,44 @@
...
+(function( $, document, wb ) {
+	&quot;use strict&quot;;
+	$('#mycontainer').on( &quot;wb-contentupdated&quot;, function( event, data ){
+		// &quot;data.ajax-type&quot; contains the insersion method [after, append, before, prepend, replace]
+		// &quot;data.content&quot; contains the
+		$(event.currentTarget)
+			.find( wb.allSelectors )
+				.addClass( &quot;wb-init&quot; )
+				.filter( &quot;:not(#&quot; + $elm.attr( &quot;id&quot; ) + &quot; .wb-init .wb-init)&quot; )
+					.trigger( &quot;timerpoke.wb&quot; );
```
Comment (LaurentGoderre) [inline site/pages/docs/ref/data-ajax/data-ajax-en.hbs]: @duboisp  Did you test this?

**Comment by LaurentGoderre** [comment]
Can you split this into two PR. I can merge the part about the event right away. I'm not sure anout the other part though

File: site/pages/docs/ref/data-ajax/data-ajax-en.hbs (hunk 1 above)
Comment (duboisp) [inline site/pages/docs/ref/data-ajax/data-ajax-en.hbs]: I forgot to declare ```$elm``` in this example, I will fix my PR.

**Comment by duboisp** [comment]
@LaurentGoderre do you mean the code sample and the events documentation?

**Comment by LaurentGoderre** [comment]
One PR for the event and one PR for documenting how to enhance the ajax

**Comment by duboisp** [comment]
Which one you are unsure about?

**Comment by LaurentGoderre** [comment]
the enhancing WET. The other one is a a no brainer

**Comment by duboisp** [comment]
That is just a code snippet that show one possibility. I do not understand the risk or the issue.

**Comment by LaurentGoderre** [comment]
I'd have to test to make sure it works before merging

**Comment by duboisp** [comment]
@LaurentGoderre @MaximMartel the snipped is a variation of what it is currently used by the WET core. see: https://github.com/wet-boew/wet-boew/blob/master/src/core/wb.js#L204

**Comment by LaurentGoderre** [comment]
eeh, we can always improve it later if need be

**Comment by duboisp** [comment]
Thanks @LaurentGoderre for merging. Feel free to take the time to submit a PR that improve it.

**Pull request merged**